app/presenters/prior_authority/tasks/further_information.rb

```diff
@@ -0,0 +1,35 @@
+module PriorAuthority
+  module Tasks
+    class FurtherInformation < Base
+      PREVIOUS_TASKS = [
+        PriorAuthority::Tasks::Ufn,
+        PriorAuthority::Tasks::CaseContact,
+        PriorAuthority::Tasks::ClientDetail,
+        PriorAuthority::Tasks::CaseAndHearingDetail,
+        PriorAuthority::Tasks::PrimaryQuote,
+        PriorAuthority::Tasks::AlternativeQuotes,
+        PriorAuthority::Tasks::ReasonWhy,
+      ].freeze
+      FORM = ::PriorAuthority::Steps::FurtherInformationForm
+
+      def path
+        edit_prior_authority_steps_further_information_path
+      end
+
+      def completed?
+        FORM.build(record, application:).valid?
+      end
+
+      def can_start?
+        PREVIOUS_TASKS.all? { |task| fulfilled?(task) }
```

Comment:
I _think_ this could just be `true`, as you'll never see this card if you're not allowed to start it.

Reply:
I think you're right now that I think about it